src/briefcase/commands/new.py

```diff
@@ -9,11 +9,25 @@
 from briefcase.config import PEP508_NAME_RE
 from briefcase.exceptions import NetworkFailure
 
+from keyword import iskeyword
+
 from .base import BaseCommand, BriefcaseCommandError
 from .create import InvalidTemplateRepository
 
 VALID_BUNDLE_RE = re.compile(r'[a-zA-Z0-9-]+(\.[a-zA-Z0-9-]+)+$')
 
+javareserved = [
```

Comment:
A really minor thing - but this should probably be a set; it's a micro-optimisation in this particular case, but it's a good habit to use datastructures that match the way in which they will be used. In this case, we want membership, and don't care about order, so set is a much better fit.